cmd/build/command.go

```diff
@@ -60,14 +65,29 @@ func Build(ctx context.Context) *cobra.Command {
 			options.CommandArgs = args
 			bc := NewBuildCommand()
 
+			// The context must be loaded before the read manifest. Otherwise,
+			// secrets will not be resolved when GetManifest is called and
+			// the manifest will load empty values.
+			if err := bc.loadContext(ctx, options); err != nil {
+				return err
+			}
+
 			builder, err := bc.getBuilder(options)
 			if err != nil {
 				return err
 			}
 
-			if err := builder.LoadContext(ctx, options); err != nil {
-				return err
+			if _, ok := builder.(*buildv1.OktetoBuilder); ok {
+				maxV1Args := 1
+				docsURL := "https://okteto.com/docs/reference/cli/#build"
```

Comment:
Move it as constant

Reply:
if it is used only once I think we could add both values directly to the error string, wdyt?

Reply:
I don't think it should be on the error string, changing the docs url is quite difficult now that we have them all as variables with the suffix `URL` so having them inside the string can make it harder when updating docs

Reply:
yes, agree